src/model.rs

```diff
@@ -119,6 +119,34 @@ pub struct Collections {
 }
 
 impl Collections {
+    /// Remove associated schedules with route points
+    pub fn remove_route_points(&mut self) {
+        let vj_idxs: Vec<Idx<VehicleJourney>> =
+            self.vehicle_journeys.iter().map(|(idx, _)| idx).collect();
+        let is_point_of_route = |stop_time: &StopTime| -> bool {
```

Comment:
If the function above is named `remove_route_points`, this closure can probably be named `is_route_point` :wink:

Reply:
done in f63f8fd7b66e8aea900bc20513d580610d6e75f8